app/assets/stylesheets/views/_history-buildings.scss

```diff
@@ -0,0 +1,41 @@
+.histories-show {
+  .app-figure {
```

Comment:
We have a [figure component](https://government-f-msw-10-dow-u2ibe2.herokuapp.com/component-guide/figure) we could use instead of adding custom styles.